teerex/src/lib.rs

```diff
@@ -106,6 +109,24 @@ pub mod pallet {
 	#[pallet::getter(fn confirmed_calls)]
 	pub type ExecutedCalls<T: Config> = StorageMap<_, Blake2_128Concat, H256, u64, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn latest_sidechain_block_header)]
+	pub type LatestSidechainBlockHeader<T: Config> =
+		StorageMap<_, Blake2_128Concat, ShardIdentifier, Header, ValueQuery>;
+
+	pub type ShardBlockNumber = (ShardIdentifier, u64);
```

Comment:
Can you move the type defintion above `LatestSidechainBlockHeader` ? Substrate convention is that storage, calls and such are always grouped together, which makes the large files a little less messy.

Reply:
more precise? above the pallet definition or inside? here: https://github.com/integritee-network/pallets/blob/9b7528f2d287129dc7e127475b195a3ac91ca060/teerex/src/lib.rs#L41?

Reply:
Hm. I don't care that much, but seems to be convention to have non-runtime related types outside the pallet. So yes, just like in the teerex pallet.